core/src/main/kotlin/com/gitlab/kordlib/core/entity/User.kt

```diff
@@ -115,25 +120,21 @@ open class User(
         suspend fun getDefaultImage(): Image = Image.fromUrl(kord.resources.httpClient, defaultUrl)
 
         /**
-         * Requests to get the avatar of the user as an [Image], or returns null if the format is not supported.
+         * Requests to get the avatar of the user as an [Image] given [format] and [size], or returns null if the
+         * format is not supported.
          *
-         * @param format the requested image format, defaults to the behavior of [url] if null.
+         * @param format    The requested image format, defaults to the behavior of [format] if not specified.
+         * @param size      The requested image resolution, defaults to [Image.Size.SIZE_128] if not specified.
          */
-        suspend fun getImage(format: Image.Format): Image? {
-            val url = getUrl(format) ?: return null
+        suspend fun getImage(format: Image.Format = this.format, size: Image.Size = Image.Size.SIZE_128): Image? {
+            val url = getUrl(format, size) ?: return null
 
             return Image.fromUrl(kord.resources.httpClient, url)
         }
 
-        /**
-         * Requests to get the avatar of the user as an [Image], prioritizing gif for animated avatars and png for others.
-         */
-        suspend fun getImage(): Image = Image.fromUrl(kord.resources.httpClient, url)
```

Comment:
Why remove this?

Reply:
Satisfied by optional argument of format, in the function above. So no extra need of method overload.

Reply:
It is not, this function's return type is not nullable, while the one with optional arguments is.